prog/weavewait/main.go

```diff
@@ -26,6 +26,11 @@ func main() {
 		args = args[1:]
 	}
 
+	if len(args) > 0 && args[0] == "-c" { // tell pid 1 to continue
+		checkErr(syscall.Kill(1, syscall.SIGUSR2))
```

Comment:
It is only pid 1 if the container has its own process namespace. Which won't be the case if it was started with `--pid=host`. That will be uncommon, but still something we should handle.

Also, I do wonder how safe it is to assume it is pid 1 even when the container *does* have its own process namespace. Does docker guarantee that a) in the docs, or, at least, b) in the code?